modules/apps/frontend-js/frontend-js-jquery-web/src/main/resources/META-INF/resources/jquery/form.js

```diff
@@ -1076,8 +1084,12 @@
 				var parseJSON =
 					$.parseJSON ||
 					function (s) {
-						/*jslint evil:true */
-						return window['eval']('(' + s + ')');
+
+						// Arise an error resolvable including jquery instead of
+						// making a new function using unsanitized inputs
+
+						window.console.error('jquery.parseJSON is undefined');
+						return null;
```

Comment:
Since we're patching manually now, we will want an extra line here before the `return`.

What is "an error resolvable including jquery?" I don't think the message is really clear so I would probably just remove it completely. If we want to keep something, I'd keep the `Throw an error instead of making a new function using unsanitized inputs to avoid XSS attacks`

Reply:
done